modules/pytket-honeywell/setup.py

```diff
@@ -40,9 +40,11 @@
     install_requires=[
         "pytket ~= 0.11.0",
         "requests >= 2.2",
+        "types-requests",
         "websockets >= 7.0",
         "nest_asyncio >= 1.2",
         "pyjwt >= 1.7, < 2.0",
+        "types-jwt",
```

Comment:
Looks good, although we'll need to remember to remove this should we use pyjwt 2.0+, according to https://pypi.org/project/types-jwt/.

Reply:
Yes, good point.